docs_source_files/config.toml

```diff
@@ -224,3 +224,35 @@ weight = 20
 name = "<i class='fas fa-fw fa-bullhorn'></i> 貢献方法"
 url = "contributing"
 weight = 30
+
+[Languages.ko]
+title = "Selenium 문서"
+weight = 6
```

Comment:
Language `ja` has already weight 6, can you please change it to 7?

Reply:
Thanks. changed it to 7